docs/install-olm.md

```diff
@@ -0,0 +1,64 @@
+# How do I install OLM?
+
+The OLM is installable on Kubernetes clusters. It is already installed and available by default on 4.x OpenShift 
```

Comment:
This section of doc is for upstream so in my opinion, we shouldn't mention any OpenShift-specific in here. OpenShift-related doc should be under `/doc` directory. Also, when you mention OpenShift cluster, perhaps it would be beneficial to add a link to OpenShift installer repo somewhere.

Reply:
+1